<launch>
	<!-- This launch file starts, segmentation, recognition. and the database communication node. -->
	<arg name="flatten_table" default="false"/>	
	<arg name="tabletop_segmentation_points_input" default="camera/depth/points" />

	<!-- segmentation -->
	<include file="$(find tabletop)/launch/tabletop_segmentation.launch">
	  <arg name="tabletop_segmentation_points_in" value="$(arg tabletop_segmentation_points_input)" />
	  <arg name="flatten_table" value="$(arg flatten_table)"/>
	</include>

	<!-- object detection -->
	<node pkg="tabletop" name="tabletop_object_detection" type="tabletop_object_detection" respawn="true" output="screen">
		<!--topic remapping-->
	        <remap from="markers_out" to="tabletop_object_detection_markers" />
    </node>
    
</launch>


